app/src/main/java/io/appium/uiautomator2/server/AppiumServlet.java

```diff
@@ -116,6 +117,7 @@ private void registerDeleteHandler() {
     private void registerPostHandler() {
         register(postHandler, new NewSession("/wd/hub/session"));
         register(postHandler, new FindElement("/wd/hub/session/:sessionId/element"));
+        register(postHandler, new ActiveElement("/wd/hub/session/:sessionId/element/active"));
```

Comment:
the handler must be changed to GET, please create a PR with a patch

Reply:
Thanks for keeping us honest @mykola-mokhnach -- per our offline conversation we'll get it done.

Reply:
This has been resolved with: https://github.com/appium/appium-uiautomator2-server/pull/424